crates/iroha_data_model/src/account.rs

```diff
@@ -77,6 +77,24 @@ mod model {
         pub metadata: Metadata,
     }
 
+    /// Read-only reference to [`Account`].
+    /// Used in query filters to avoid copying.
+    pub struct AccountEntry<'world> {
+        /// Identification of the [`Account`].
+        pub id: &'world AccountId,
+        /// Metadata of this account as a key-value store.
+        pub metadata: &'world Metadata,
+    }
+
+    /// [`Account`] without `id`.
+    /// Needed only for [`World::accounts`] map to reduce memory usage.
+    /// In other places use [`Account`] directly.
+    #[derive(Clone, Deserialize, Serialize)]
+    pub struct AccountValue {
+        /// Metadata of this account as a key-value store.
+        pub metadata: Metadata,
+    }
```

Comment:
Is it necessary to define it under the `#[model]` procmacro? _(applies to other new `*Value` structs)_

Reply:
Moved out of `#[model]` macro